File: ruby_101/conditional.rb
def get_number_from_somewhere; 2 end

# START:conditional
# Ruby:
puts case get_number_from_somewhere
     when 2 then 'twins'
     when 3 then 'triplets'
     when 4 then 'quadruplets'
     else        'unknown'
     end
# END:conditional

File: ruby_101/Conditional.java
public class Conditional {
    private int getNumberFromSomewhere() {
        return 2;
    }

    public void printGroupName() {
        // START:conditional
        // Java:
        String result = "";

        switch (getNumberFromSomewhere()) {
        case 2:  result = "twins";       break;
        case 3:  result = "triplets";    break;
        case 4:  result = "quadruplets"; break;
        default: result = "unknown";     break;
        }

        System.out.println(result);
        // END:conditional
    }
}